build.gradle

```diff
@@ -70,6 +70,7 @@ allprojects {
         compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.7'
 
         testCompile group: 'junit', name: 'junit', version: '4.11'
+        testCompile group: 'com.tngtech.java', name: 'junit-dataprovider', version: '1.10.0'
```

Comment:
Why is this dep added?

Reply:
I needed parameterized test. (https://github.com/allegro/hermes/pull/1083/files#diff-d6b07aceab798e7a80aff1c6ca7e420dR253)